src/routes/files.ts

```diff
@@ -22,17 +22,41 @@ export default async function files(fastify: FastifyInstance) {
 
   fastify.get('/files/credentials', async (req) => {
     const sessionCache: SessionCache = req.sessionCache;
+    const results = new Map<ContactType, String[][]>();
     const places = sessionCache.getPlaces();
-    const refinedRecords = transform(places, (place: Place) => {
-      return [
-        place.type.friendly,
+    places.forEach(place => {
+      const parent = Config.getParentProperty(place.type);
+      const record = [
+        place.hierarchyProperties[parent.property_name],
         place.name,
         place.creationDetails.username,
```

Comment:
@kennsippell what do you think about adding the contact's name and phone number to the list? Would that be enough for cases where places share names?

Reply:
I'd probably prioritize having the parent's name (eg CHU or sub-county). That seems super useful for sorting, distribution, and other activities.

I dunno if the contact's name and phone number seems valuable. I have no objections to adding it.

Reply:
Yes i include the place's parent as the first column in the credential list. The problem is for the CHP lists, the users expect both CHU and Subcounty in the credential list and i was trying to give them more ways to identify the users

Reply:
> I don't think it is always on the contact.

I thought `phone` was only available on the contact model? Do you know when this isn't the case?

Reply:
For Kenya phone is only on the contact, ya.  But it isn't required, and you can put the phone attribute on the place just as easily. 

It is just like any other attribute, so why assume it is only on the contact?